### PR TITLE
Improve FT transfer FE

### PR DESCRIPTION
**Pull request by pivilartisant**

### CODE REVIEW

File: web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx [hunk 1]
```diff
@@ -166,7 +166,7 @@ export default function Advanced(props: AdvancedProps) {
               placeholder={Intl.t('send-coins.custom-fees')}
               name="fees"
               variant="MAS"
-              value={feesField}
+              value={(customFees && currentFees) || feesField}
```
Comment (Thykof) [inline web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx:169]: what is the logic here? if customFees === bool then currentFees else feesField?

or 

if (customFees === bool and currentFees !== '') then currentFees else feesField

this syntax `customFees && currentFees) || feesField` is not really clear to me.

File: web-frontend/src/pages/TransferCoins/SendCoins/SendForm.tsx [hunk 2]
```diff
@@ -61,9 +61,11 @@ export function SendForm(props: SendFormProps) {
   );
   const [fees, setFees] = useState<string>(PRESET_LOW);
   const [recipient, setRecipient] = useState<string>(
-    data?.recipientAddress || '',
+    (data && data.recipientAddress) || '',
```
Comment (Thykof) [inline web-frontend/src/pages/TransferCoins/SendCoins/SendForm.tsx:64]: same question here

File: web-frontend/src/pages/TransferCoins/SendCoins/SendForm.tsx [hunk 3]
```diff
@@ -61,9 +61,11 @@ export function SendForm(props: SendFormProps) {
...
+  );
+  const [selectedAsset, setSelectedAsset] = useState<Asset | undefined>(
+    (data && data.asset) || undefined,
```
Comment (Thykof) [inline web-frontend/src/pages/TransferCoins/SendCoins/SendForm.tsx:67]: same question here

**Review by Thykof — COMMENTED** [review]
super!

File: web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx (hunk 1 above)
Comment (pivilartisant) [reply web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx]: I pushed a modification of the syntax. Indeed : 
```customFees === bool then currentFees else feesField?```

File: web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx [hunk 4]
```diff
@@ -48,12 +48,13 @@ export default function Advanced(props: AdvancedProps) {
 
   const [error, setError] = useState<InputsErrors | null>(null);
   const [fees, setFees] = useState<string>(initialFees);
-  const [feesField, setFeesField] = useState<string>('');
-  const [customFees, setCustomFees] = useState<boolean>(initialCustomFees);
+  const [customFees, setAdvancedFees] = useState<string>('');
```
Comment (Thykof) [inline web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx:51]: the good practice would be customFees, setCustoFees or advancedFees, setAdvancedFees

**Review by Thykof — APPROVED** [review]

**Review by Thykof — CHANGES_REQUESTED** [review]
QA KO: click on advanced, select custom, select 0.5, validate, click on advancde, validate, go to send and see that the value custom fee is not taken

**Pull request merged**